packages/common/test/util/http.test.js

```diff
@@ -81,6 +93,22 @@ describe('parseUrl', () => {
       expect(e.message).to.eql('Invalid URL');
```

Comment:
Looks like we are checking if the url is invalid and not the checking if query parameters are extracted correctly 
Perhaps we should update the test name ?

Reply:
I think you're looking at a collapsed diff here. `should extract query parameters` and `should throw if path and no base` are two different tests